bin/word-tokenization-benchmark

```diff
@@ -1,121 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
- [ ] Add bin/word-tokenization-benchmark

Reply:
That was my mistake, I already fixed it. 😥